tests/tooling/page.py

```diff
@@ -0,0 +1,7 @@
+from playwright.async_api._generated import Page
+
+
+async def page_stable(page: Page) -> None:
+    """Only return when network is idle and DOM has loaded"""
+    await page.wait_for_load_state("networkidle")
+    await page.wait_for_load_state("domcontentloaded")
```

Comment:
I don't expect `reactpy_router` to have many test tooling functions, so we should move this content into a `tests>utils.py` until there is a significant need to start splitting into smaller modules.